client/my-sites/term-selector/no-results.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import { localize } from 'i18n-calypso';
+
+class TermSelectorNoResults extends PureComponent {
```

Comment:
You've mixed `React.createClass` and `Component` classes in this pull request. Any particular reason for this?

Related: p1466013240000130-slack-calypso-framework